sdk/lib/opentelemetry/sdk/trace/export/batch_span_processor.rb

```diff
@@ -25,20 +27,23 @@ module Export
         # export fails with {FAILED_RETRYABLE}, backing off linearly in 100ms
         # increments.
         class BatchSpanProcessor
+          EXPORT_TIMEOUT_MILLIS = 30_000
           SCHEDULE_DELAY_MILLIS = 5
```

Comment:
While you're in here, this should be `5_000` instead of `5`.

Reply:
:heavy_check_mark: